bzip2-sys/lib.rs

```diff
@@ -69,6 +69,6 @@ abi_compat! {
 }
 
 #[no_mangle]
-pub fn bz_internal_error(errcode: c_int) {
+pub extern "C" fn bz_internal_error(errcode: c_int) {
     panic!("bz internal error: {}", errcode);
 }
```

Comment:
do we need to use `extern "system"` on windows given our MSRV?

Reply:
`extern "system"` is the same as `extern "C"` except on 32bit Windows where it is the same as `extern "stdcall"`. bzip2 uses `extern "stdcall"` for all `BZ2_*` methods on 32bit Windows, but uses the regular C ABI for `bz_internal_error`.